src/daemon/daemon.cpp

```diff
@@ -1928,11 +1887,7 @@ try // clang-format on
             if (instance->current_state() == VirtualMachine::State::delayed_shutdown)
                 delayed_shutdown_instances.erase(name);
 
-            for (const auto& mount_handler : config->mount_handlers)
-            {
-                mount_handler.second->stop_all_mounts_for_instance(name);
-            }
-
+            mounts[name].clear();
```

Comment:
No stopping needed, to avoid corrupting data on the host? In any implementation?

Reply:
Hmm, even when deleting the instance? I know we discussed this during a daily sync, and from what I understood, we should prevent only stopping and suspending from happening in case of unmount failure. Should we also prevent non-purge deleting?

Reply:
Right, we did discuss that. OK, let's stick to what we said then. I guess corruptions could only arise if the mount was still being written to, which the user would have to ensure was not the case before deleting. I just don't know when writes are flushed...